device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,140 @@
+          $ref: '#/components/schemas/Device.Dhcpv6client.DuidType'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.IaType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Identity Association: Each IA has an associated IAID. IA_NA and IA_TA options represent different types of 
+            IPv6 addresses and parameters accepted by DHCPv6 clients each used in different context by an IPv6 node. IA_NA 
+            is the Identity Association for Non-temporary Addresses option. IA_TA is the Identity Association for Temporary 
+            Addresses option. IA_PD and IA_NAPD options represent one or more IPv6 prefix and parameters. IA_PD is the Identity 
+            Association for Prefix Delegation and IA_NAPD s the Identity Association for Temporary Prefix Delegation.
+          type: string
+          default: iana
+          x-field-uid: 1
+          x-enum:
+            iana:
+              x-field-uid: 1
+            iata:
+              x-field-uid: 2
+            iapd:
+              x-field-uid: 3
+            ianapd:
+              x-field-uid: 4
+        iana:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 2 
+        iapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 3
+        ianapd:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaTimeValue'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.IaTimeValue:
+      description: >-
+        The container for the suggested times at which the client contacts the server or any available server.
+      type: object
+      properties:
+        ta_1:
+          description: >-
+            The suggested time at which the client contacts the server from which the addresses were obtained to 
+            extend the lifetimes of the addresses assigned.
+          type: integer
+          format: uint64
+          maximum: 2147483647
+          default: 302400
+          x-field-uid: 1
+        ta_2:
+          description: >-
+            The suggested time at which the client contacts any available server to extend the lifetimes of the addresses assigned.
+          type: integer
+          format: uint64
+          maximum: 2147483647
+          default: 483840
+          x-field-uid: 2
+
+    Device.Dhcpv6client.DuidType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Each DHCP client and server has a DUID. DHCP clients use DUIDs to identify a server in messages where a server 
+            .needs to be identified.
+          type: string
+          default: llt
+          x-field-uid: 1
+          x-enum:
+            llt:
+              x-field-uid: 1
+            en:
+              x-field-uid: 2
+            ll:
+              x-field-uid: 3
+        llt:
+          $ref: '#/components/schemas/Device.Dhcpv6client.NoDuid'
+          x-field-uid: 2
+        em:
```

Comment:
It should be `en`  as description said `The container for the DUID-EN`

Reply:
updated